### PR TITLE
Badge: VR fixes

### PR DESCRIPTION
**Pull request by AlbertCarreras**
Badge: VR fixes

### CODE REVIEW

File: packages/gestalt/src/Badge.css [hunk 1]
```diff
@@ -3,10 +3,17 @@
   composes: rounding1 from "./Borders.css";
   align-content: center;
   display: flex;
-  padding: var(--space-0) 6px;
```
Comment (rlingineni) [inline packages/gestalt/src/Badge.css:6]: I think 6px should be a component token as `--badge-padding-left`

File: packages/gestalt/src/Badge.css [hunk 2]
```diff
@@ -3,10 +3,17 @@
...
   white-space: nowrap;
 }
 
+.padding {
+  padding: var(--space-0) 6px;
```
Comment (rlingineni) [inline packages/gestalt/src/Badge.css:10]: I think 6px should be a component token as `--badge-padding-y` or something, so we can change it easier int he future

File: packages/gestalt/src/Badge.css [hunk 3]
```diff
@@ -3,10 +3,17 @@
...
+}
+
+.paddingVR {
+  padding: var(--sema-space-0) var(--sema-space-100);
```
Comment (rlingineni) [inline packages/gestalt/src/Badge.css:14]: Ideally we use component tokens for these

File: packages/gestalt/src/Badge.css (hunk 2 above)
Comment (rlingineni) [reply packages/gestalt/src/Badge.css:10]: Oh it seems like this is classic

File: packages/gestalt/src/icons-vr-theme/compact/index.ts [hunk 4]
```diff
@@ -10,6 +10,7 @@ import compactChevronUp from './compact-chevron-up.svg';
 import compactDash from './compact-dash.svg';
 import compactInfoCircleFill from './compact-info-circle-fill.svg';
 import compactLock from './compact-lock.svg';
+import compactSparkle from './compact-sparkle.svg';
```
Comment (rlingineni) [inline packages/gestalt/src/icons-vr-theme/compact/index.ts:13]: If it's not too much - could you also paste this into the Android package as well. Thank you

**Review by rlingineni — APPROVED** [review]
comments

File: packages/gestalt/src/Badge.css (hunk 3 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/Badge.css:14]: we don't have cmp token for spacing yet.. i think

File: packages/gestalt/src/Badge.css (hunk 1 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/Badge.css:6]: we don't have cmp token for spacing yet.. i think

**Pull request merged**